docs/reference/arm.md

```diff
@@ -34,7 +34,6 @@ run as expected.
 
 ## Building Redis in the Pi
 
-* Download Redis version 4.0 or higher.
```

Comment:
It should tell users to download the software.

Reply:
updated the diff, Thanks!